content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -779,6 +780,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `HyperVContainer`: Enable
   [Hyper-V isolation](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container)
   for Windows containers.
+- `IdentifyPodOS`: Allows the Pod OS field to be specified. This helps in identifying the OS of the pod
+  authoritatively during the API server admission time. In Kubernetes {{< skew currentVersion >}}, the allowed values for the `Name` field in `OS` field of the pod spec are `windows` and `linux`.
```

Comment:
I presume this is `.spec.os.name`?

Reply:
Yes, would it be helpful if mention directly `.spec.os.name`?

Reply:
Definitely - readers don't need to know what those field names look like in the source code.